Author: <agmartin@debian.org>
Description: Provide a version function and variable, so we know about the original snapshot.

Index: dictionaries-common/support/emacsen/flyspell.el
===================================================================
--- dictionaries-common.orig/support/emacsen/flyspell.el	2014-10-14 18:27:35.875490044 +0200
+++ dictionaries-common/support/emacsen/flyspell.el	2014-10-14 18:27:35.871490054 +0200
@@ -36,6 +36,26 @@
 ;; Some user variables control the behavior of flyspell.  They are
 ;; those defined under the `User variables' comment.
 
+;; --------------------------------------------------------------------
+;; This file has been modified by Agustin Martin <agmartin@debian.org>
+;; for the Debian GNU/Linux system to meet the requirements of the Debian
+;; Spelling Policy, add capabilities and fix bugs
+;;
+;; This file is included in the Debian dictionaries-common package
+;;
+;; We define flyspell-version function and var to let people know about this
+
+(defun flyspell-version ()
+  "The flyspell version"
+  (interactive)
+  "FSF emacs CVS20090625(r1.145) + Debian `dictionaries-common' changes (like XEmacs compatibility)")
+
+;; ispell.el provides a var for this. Provide one too.
+(defvar flyspell-version (flyspell-version)
+  "The flyspell version")
+
+;; --------------------------------------------------------------------
+
 ;;; Code:
 
 (require 'ispell)
